### ISSUE TITLE
docbug: geoip parser: there is no documentation

### ISSUE THREAD

**Issue by faxm0dem** [balabit/syslog-ng#648]
I managed to make the lonlat lookup work by doing:
```
wget -O- http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz | gunzip -c - > /usr/share/GeoIP.dat
```
Is it possible to tell the API to use a different file?
I'd like to not overwrite the one provided by the `geoip-database` package provided by debian

**Comment by faxm0dem** [comment]
it seems the right way to manage the `GeoIP.dat` file is using `geoipupdate`.
I'll change the subject of this issue to: "add documentation for geoip" :)

**Comment by faxm0dem** [comment]
There seems to be a problem when trying to use the parser and the template function at the same time. The former expects the City db, while the latter requires the Country db.
So it seems not possible to use both at the same time.

**Comment by fekete-robert** [comment]
@lbudai Solved in #662 , please close this issue
